src/components/PaymentInformationForm/index.js

```diff
@@ -0,0 +1,114 @@
+import { useState } from "react";
+import {
+  Main,
+  Choices,
+  Option,
+  Accommodation,
+} from "../PaymentInformationForm/PaymentWrapper";
+import AccomodationFinishMessage from "./AccomodationFinishMessage";
+
+export default function PaymentInformationForm() {
+  const [isSelected, setIsSelected] = useState({
+    isOnline: null,
+    price: null,
+    hashotelReservation: null,
+    hotelPrice: null,
+  });
+  const [isPresential, setIsPresential] = useState(false);
+  const [isOnlineOption, setIsOnlineOption] = useState(false);
+  const [hasHotel, setHasHotel] = useState(false);
+  const [hasntHotel, setHasntHotel] = useState(false);
+
+  function handleModalityChoice(isOnline, price) {
+    setIsSelected({
+      isOnline,
+      price,
+      hashotelReservation: null,
+      hotelPrice: null,
+    });
+    if (isOnline === false) {
+      setIsPresential(true);
+      setIsOnlineOption(false);
+      setHasntHotel(false);
+      setHasHotel(false);
```

Comment:
Se não for online, ele ainda vai poder escolher se vai querer hotel ou não

Reply:
N entendi esse comentário. Essa mudança de váriavel é para reiniciar caso ele altere de online para presencial fazendo reinciar os estados para as escolhas iniciais

Reply:
Numa visão geral, a variável hasHotel não deveria representar se o cara escolheu um hotel ou não?